### DISCUSSION TITLE
Offline Create React App

### DISCUSSION THREAD

**Discussion by lmborione** [iTwin/itwinjs-core#759] [Q&A] [answered]
Hello,

How do you open an offline snapshot with @bentley/itwin-viewer-react Viewer ?

I try:
```
<Viewer
      snapshotPath="C:\\Users\\<username>\\AppData\\Local\\iModelJs\\imodels\\MyFile.bim"
      authConfig={{ oidcClient: AuthorizationClient.oidcClient }}
/>
```
And I get an error :  "An Error Has Occurred - TypeError: Failed to fetch"
Thanks

**Comment by calebmshafer** [comment, marked as answer]
Hi @lmborione, the `snapshotPath` option only works in the case of writing a Desktop app and you'll need to configure a couple more options to get started in that case.  If you check out the [Desktop](https://github.com/imodeljs/itwin-viewer/tree/master/packages/modules/itwin-viewer-react#desktop) section in the Viewer readme, you'll want to follow the steps to set `DesktopApp` and `authConfig`.

Note: We're working on a sample app that shows how to perform all of the setup for the Viewer which should be released in the next few weeks.
